var/spack/repos/builtin/packages/saga-gis/package.py

```diff
@@ -17,7 +17,10 @@ class SagaGis(AutotoolsPackage):
     git         = "git://git.code.sf.net/p/saga-gis/code"
 
     version('develop',  branch='master')
-    version('7.0.0',    branch='release-7.0.0', preferred=True)
+    version('7.3.0',    branch='release-7.3.0', preferred=True)
```

Comment:
Just a minor nitpick: `preferred` should not be necessary because develop is never chosen as the preferred version.

Reply:
Ok, good to know. Just used it because it was already there.